craftitems.lua

```diff
@@ -1,89 +1,89 @@
 minetest.register_craftitem("magic_materials:arcanite_crystal", {
-    description = "Arcanite Crystal",
+    description = magic_materials.S("Arcanite Crystal"),
```

Comment:
Please use `S()` when calling the translation API to reduce repetition.
```lua
local S = magic_materials.S
-- ......
description = S("str")
```

Reply:
Corrected

Reply:
Translations also